### DISCUSSION TITLE
Exporting web content to an epub file by one click

### DISCUSSION THREAD

**Discussion by mingoc** [plateaukao/einkbro#265] [Ideas]
Hi,

I love Einkbro. Could you make it possible to exporting web content to an epub file just by one click?
Currently, exporting web content to an epub requiring too many steps. Many times I want to export it more quickly: export and open it on another App to reading. I don't care the file name or chapters in the book.

Thanks and regards
Mingo

**Comment by plateaukao** [comment]
Hi, @mingoc 
It's the limitation of Android platform. It's difficult to share the saved epub file to other apps if it needs to be saved in a public folder. I need to get the file name from user first, or else, I need to show up a permission dialog to ask users to allow me to have full access right of the storage.
